lib/active_model/serializer/adapter/json.rb

```diff
@@ -1,47 +1,128 @@
 class ActiveModel::Serializer::Adapter::Json < ActiveModel::Serializer::Adapter
-        extend ActiveSupport::Autoload
```

Comment:
I totally agree it looks weird, but it makes the diff so much easier, and I do intend to get it back to sane soon enough